src/ir_Panasonic.cpp

```diff
@@ -183,3 +201,507 @@ bool IRrecv::decodePanasonic(decode_results *results, uint16_t nbits,
   return true;
 }
 #endif  // (DECODE_PANASONIC || DECODE_DENON)
+
+#if SEND_PANASONIC_AC
+// Send a Panasonic A/C message.
+//
+// Args:
+//   data:   Contents of the message to be sent. (Guessing MSBF order)
+//   nbits:  Nr. of bits of data to be sent. Typically kPanasonicAcBits.
+//   repeat: Nr. of additional times the message is to be sent.
+//
+// Status: Beta / Needs testing against a real device.
```

Comment:
@pandemixxx just confirmed it works. Should we update this one here?

Reply:
Thanks for the reminder. Fixed.